helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -258,7 +258,12 @@ agent:
       # This value is being used as a source for default memory_limiter processor configurations
       memory: 500Mi
 
-  securityContext: {}
+  # To collect container logs and journald logs, it will run the agent as a root user.
+  # To run it as non root user, uncomment below `securityContext` options.
```

Comment:
Let's add the a note about the init container. That would likely be the reason why users won't change the default.
Something like:

Setting `runAsUser` and `runAsGroup` to a non root user enables an init container that patches group permissions of container logs directories on the host filesystem to make logs readable by this non root user.